config/test.exs

```diff
@@ -2,3 +2,5 @@ use Mix.Config
 
 config :elix, Elix.Robot,
   adapter: Hedwig.Adapters.Test
+
+config :elix, :giphy_client, Elix.GiphyClientMock
```

Comment:
Ah, [this SO comment](http://stackoverflow.com/questions/30652439/importing-test-code-in-elixir-unit-test#comment49371142_30652675) (Thanks José!) pointed me to [how Phoenix requires and compiles `test/support`](https://github.com/phoenixframework/phoenix/blob/db2f953f24973fab8aa2aab3bd8fc9c87a17adc8/installer/templates/new/mix.exs#L10).